app/scripts/components/common/page-header/index.tsx

```diff
@@ -1,340 +1,80 @@
-  ${media.largeUp`
-    display: none;
-  `}
-`;
-
-export const GlobalNavToggle = styled(Button)`
-  z-index: 2000;
-`;
-
-const GlobalNavBody = styled.div`
-  display: flex;
-  flex: 1;
-
-  .shadow-top {
-    background: linear-gradient(
-      to top,
-      ${themeVal('color.primary-600')}00 0%,
-      ${themeVal('color.primary-600')} 100%
-    );
-  }
-
-  .shadow-bottom {
-    background: linear-gradient(
-      to bottom,
-      ${themeVal('color.primary-600')}00 0%,
-      ${themeVal('color.primary-600')} 100%
-    );
-  }
-`;
-
-const GlobalNavBodyInner = styled.div`
-  display: flex;
-  flex-direction: column;
-  flex: 1;
-  gap: ${variableGlsp()};
-  padding: ${variableGlsp(1, 0)};
-
-  ${media.largeUp`
-    flex-direction: row;
-    justify-content: space-between;
-    padding: 0;
-  `}
-`;
-
-const NavBlock = styled.div`
-  display: flex;
-  flex-flow: column nowrap;
-  gap: ${glsp(0.25)};
-
-  ${media.largeUp`
-    flex-direction: row;
-    align-items: center;
-    gap: ${glsp(1.5)};
-  `}
-`;
-
-const SROnly = styled.a`
-  height: 1px;
-  left: -10000px;
-  overflow: hidden;
-  position: absolute;
-  top: auto;
-  width: 1px;
-  color: ${themeVal('color.link')};
-  &:focus {
-    top: 0;
-    left: 0;
-    background-color: ${themeVal('color.surface')};
-    padding: ${glsp(0.25)};
-    height: auto;
-    width: auto;
-  }
-`;
-
-const SectionsNavBlock = styled(NavBlock)`
-  /* styled-component */
-`;
-
-const GlobalNavBlockTitle = styled(Overline).attrs({
-  as: 'span'
-})`
-  ${visuallyHidden}
-  display: block;
-  padding: ${variableGlsp(1, 1, 0.25, 1)};
-  color: currentColor;
-  opacity: 0.64;
-
-  ${media.largeUp`
-    padding: 0;
-  `}
-`;
-
-const GlobalMenu = styled.ul`
-  ${listReset()}
-  display: flex;
-  flex-flow: column nowrap;
-  gap: ${glsp(0.5)};
-
-  ${media.largeUp`
-    flex-direction: row;
-    justify-content: flex-start;
-    align-items: center;
-    gap: ${glsp(1.5)};
-  `}
-`;
+const appTitle = process.env.APP_TITLE;
+const appVersion = process.env.APP_VERSION;
 
 interface PageHeaderProps {
   mainNavItems: NavItem[];
   subNavItems: NavItem[];
-  logo: ReactElement;
+  logo?: ReactElement;
   linkProperties: LinkProperties;
 }
 
-function PageHeader(props: PageHeaderProps) {
-  const { mainNavItems, subNavItems, logo, linkProperties } = props;
-  const { isMediumDown } = useMediaQuery();
-  const [globalNavRevealed, setGlobalNavRevealed] = useState(false);
-
-  const globalNavBodyRef = useRef<HTMLDivElement>(null);
-  // Click listener for the whole global nav body so we can close it when clicking
-  // the overlay on medium down media query.
-  const onGlobalNavClick = useCallback((e) => {
-    if (!globalNavBodyRef.current?.contains(e.target)) {
-      setGlobalNavRevealed(false);
-    }
-  }, []);
+const PageHeader: React.FC<PageHeaderProps> = ({
+  mainNavItems,
+  subNavItems,
+  logo: Logo,
+  linkProperties
+}) => {
+  const [expanded, setExpanded] = useState<boolean>(false);
```

Comment:
// TODO: Add resize listener to close mobile nav when window is resized (`setExpanded(false)`)